app/graph/state.py

```
@@ -0,0 +1,20 @@
+from typing import Optional, Any
+from typing_extensions import TypedDict
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)